tests/foreman/destructive/test_rename.py

```diff
@@ -63,19 +63,19 @@ def test_positive_rename_satellite(module_org, module_product, module_target_sat
     username = settings.server.admin_username
     password = settings.server.admin_password
     old_hostname = module_target_sat.execute('hostname').stdout.strip()
-    new_hostname = f'new-{old_hostname}'
+    new_hostname = f'{old_hostname.split(".")[0]}-changed.{".".join(old_hostname.split(".")[1:])}'
```

Comment:
```suggestion
    old_shortname, old_domain = old_hostname.split(".", 1)
    new_hostname = f'{old_shortname}-changed.{old_domain}'
```

IMHO more readable :)

Reply:
I forgot about 2nd parameter in `split()` :wink:

Reply:
Suggestion accepted.